cucumber/authenticators_gcp/features/authn_gcp.feature

```diff
@@ -0,0 +1,41 @@
+Feature: GCP Authenticator - Hosts can authenticate with GCP authenticator
+
+  In this feature we define a Google Cloud Platform authenticator in policy and perform authentication
+  with Conjur, using a host with account-service-id.
```

Comment:
i wouldn't state the `account-service-id` as we will have other tests in this feature